### PR TITLE
OPENEUROPA-3180: Fix issue with failed patch for oe_content.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -33,14 +33,15 @@
         "drupal/styleguide": "~1.0-alpha3",
         "drupal/typed_link": "~1.0",
         "drush/drush": "~9.0",
+        "openeuropa/rdf_skos": "~0.8",
```
Comment (ademarco) [inline composer.json]: We can remove this as it has been fixed on oe_content now.

File: composer.json [hunk 2]
```diff
@@ -33,14 +33,15 @@
...
         "easyrdf/easyrdf": "0.10.0-alpha.1 as 0.9.1",
         "friends-of-behat/service-container-extension": "^1.0",
         "guzzlehttp/guzzle": "~6.3",
         "nikic/php-parser": "~3.0",
         "openeuropa/behat-transformation-context": "~0.1",
         "openeuropa/code-review": "~1.1",
         "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/oe_content": "dev-master",
+        "openeuropa/oe_content": "1.8.1",
```
Comment (ademarco) [inline composer.json]: We can also discard this as that has been fixed on oe_content master now, right? ~I think this PR is not necessary anymore.~ We can update typed_link in the scope of this PR.

**Review by ademarco — CHANGES_REQUESTED** [review]

File: composer.json [hunk 3]
```diff
@@ -33,14 +33,15 @@
         "drupal/styleguide": "~1.0-alpha3",
         "drupal/typed_link": "~1.0",
```
Comment (ademarco) [inline composer.json]: We need this to be moved to ~1.1.

**Review by ademarco — CHANGES_REQUESTED** [review]

File: composer.json (hunk 3 above)
Comment (sergepavle) [reply composer.json]: Updated

File: composer.json (hunk 2 above)
Comment (sergepavle) [reply composer.json]: Done

**Review by ademarco — APPROVED** [review]

**Review by hernani — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**